pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -207,11 +207,15 @@ func (r *bindingReconciler) reconcile(ctx context.Context, apiBinding *apisv1alp
 
 	// Record the APIExport's host cluster name for lookup in webhooks.
 	// The full path is unreliable for this purpose.
-	clusterName := logicalcluster.From(apiExport)
-	apiBinding.Status.APIExportClusterName = clusterName.String()
+	apiBinding.Status.APIExportClusterName = logicalcluster.From(apiExport).String()
 
 	var needToWaitForRequeueWhenEstablished []string
 
+	checker, err := NewConflictChecker(logicalcluster.From(apiBinding), r.listAPIBindings, r.getAPIExportByPath, r.getAPIResourceSchema, r.getCRD, r.listCRDs)
```

Comment:
Any reason this is exported? This looks like internal helpers inside reconcile loop.

Reply:
fixed